src/Erl/Data/Map.purs

```diff
@@ -46,6 +48,16 @@ foreign import mapWithKeyImpl :: forall k a b. (Fn2 k a b) -> Map k a -> Map k b
 mapWithKey :: forall k a b. (k -> a -> b) -> Map k a -> Map k b
 mapWithKey f m = mapWithKeyImpl (mkFn2 f) m
 
+foreign import memberImpl :: forall k a. k -> Map k a -> Boolean
+
+member :: forall k a. k -> Map k a -> Boolean
```

Comment:
No need for a separate `memberImpl` here, it can just be `foreign import member` and export that